poetry.lock

```diff
@@ -25,7 +25,7 @@ tests-no-zope = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy
 
```

Comment:
Maybe run it with poetry 1.3? They released recently a new lock version (which is compatible with 1.2).

https://python-poetry.org/blog/announcing-poetry-1.3.0/

Also, if you renamed the PR name, maybe you should rename the commit message too?